Gemfile

```diff
@@ -47,6 +47,7 @@ gem 'foundation-rails'
 gem 'jquery-datatables-rails', git: 'https://github.com/rweng/jquery-datatables-rails.git'
 gem 'activeresource-response'
 gem 'puma'
+gem 'server_timing'
```

Comment:
do we need this in prod?

Reply:
Yeah I think it's useful for Dan to see.